### ISSUE TITLE
HA addon additional parameters for mosquitto_pub [not a issue]

### ISSUE THREAD

**Issue by reaper7** [weetmuts/wmbusmeters#350]
Is it possible to add parameters for mosquitto_pub in HA addon?
Exactly the additional RETAIN flag as an mqtt option, when configuring this add-on.

This would solve the case of unknown values when restarting the HA, while the meters are not sending information (over the weekend)

**Comment by reaper7** [comment]
@BIBOLV - it would be better if it could be written as option, hardcoded may not suit everyone.

**Comment by BIBOLV** [comment]
Yes, it makes more sense, canceled PR. Leaving for @to-masz to think about implementstion approach.

**Comment by weetmuts** [comment]
Do we need to follow up on this one, or can it be closed? :-)

**Comment by SaSa1983** [comment]
I'd like to have support for the retain parameter.
Thus upon restart of HA (e.g. after an update) the sensor value willnot fallback to "unavailable" until a packetis received, but rather stay on the previous value

**Comment by SaSa1983** [comment]
@weetmuts Is there any plan about the retain option?

**Comment by BIBOLV** [comment]
Retain flag is present in mqtt script by default - https://github.com/wmbusmeters/wmbusmeters-ha-addon/blob/main/wmbusmeters-ha-addon-edge/run.sh#L89 (-r = retain)
and it can be confirmed by looking at mqtt message itself:
![retain](https://user-images.githubusercontent.com/49994376/224562452-322d6180-f11f-481f-ab03-bd427421bc4f.PNG)

**Comment by BIBOLV** [comment]
Is this question still actual or we can close this ticket?

**Comment by SaSa1983** [comment]
As far as I know there's still no option in the current release version of the addon

**Comment by BIBOLV** [comment]
As I explained - retain flag is on by default, then you want possibility to remove it, correct?
